static/sass/styles.scss

```diff
@@ -485,12 +438,9 @@ $p-small-lh-diff: map-get($line-heights, default-text) - map-get($line-heights,
   background-color: #f3f3f3;
 }
 
-/// ochuko p. 17/05/2023
-/// can be removed once added upstream
-.p-strip--paper {
-  @extend .p-strip;
-
-  background-color: #f2f2f2;
+// Adjustment for career navigation
+.muted-color {
+  color: #a7a7a7;
 }
```

Comment:
You don't need this if you use p-strip--dark

Reply:
`p-strip` adds additional spacing that will not fit with design's spacing requirements.